src/app/Gateways/Alphas/CreateDocumentRequest.js

```diff
@@ -0,0 +1,45 @@
+import { hackneyToken } from '../../lib/Cookie';
+
+const success = requestId => ({
+  requestUrl: `${process.env.REACT_APP_DOC_UPLOAD_API_URL}/requests/${requestId}`,
+  success: true
+});
+const fail = () => ({ success: false });
+
+export default async customer => {
+  if (!customer) return fail();
+
+  try {
+    const metadata = {
+      firstName: customer.name.map(n => n.first),
+      lastName: customer.name.map(n => n.last),
```

Comment:
Assuming name is always an array, otherwise this might end up mapping `['t', 'o', 'm']`, for example if it could also be a string.

Reply:
yeah, always an array